frinx-uniconfig/user-guide/uniconfig-operations/subtree-manager/rpc_calculate-subtree-git-like-diff/readme.md

```diff
@@ -99,12 +102,11 @@ curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig
   "errors": {
     "error": [
       {
-        "error-message": "Field target-path is not specified in input request",
+        "error-type": "application",
         "error-tag": "invalid-value",
```

Comment:
probably missing parameter would be better error-tag

Reply:
Fixed in doc. My mistake.